src/main/java/seedu/address/model/person/PriorityValue.java

```diff
@@ -66,6 +66,6 @@ public static String[] getShortPriorities() {
 
     @Override
     public String toString() {
-        return this.name().toLowerCase();
+        return this.name().toUpperCase();
```

Comment:
Is displaying as uppercase the responsibility of the toString method or the ui logic?

Reply:
I think this would be better as toString's responsibility as we want the priority to always appear as CAPS :D